intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -227,6 +276,60 @@ def train(data):
 # From the above view, we can see that the runtime of ``enumerate(DataLoader)`` is reduced,
 # and the GPU utilization is increased.
 
+######################################################################
```

Comment:
Line 265: Then let’s choose the just profiled run in left "Runs" dropdown list.

Do you mean " let's choose the recently profiled run..."?

Reply:
Yes.